src/components/Table/index.tsx

```diff
@@ -0,0 +1,260 @@
+import React, { useState } from 'react';
+import PerfectScrollbar from 'react-perfect-scrollbar';
+import {
+    Box,
+    Card,
+    Table,
+    TableBody,
+    TableCell,
+    TableHead,
+    TablePagination,
+    TableRow,
+    CardHeader,
+    Divider,
+    TextField,
+    InputAdornment,
+    SvgIcon,
+    TableSortLabel,
+} from '@material-ui/core';
+import orderByFunc from 'lodash/orderBy';
+import formatDate from '../../utils/DateFormat';
+import useStyles from './style';
+import { Search as SearchIcon } from 'react-feather';
+import { genPaginationLabel, applyPagination } from '../../utils/Pagenation';
+import applyFilter from '../../utils/FilterTable';
+
+interface TableColumnHeading<T> {
```

Comment:
maybe its better to change the name to TableColumn

Reply:
yes, sure